src/main/java/com/epam/lowcost/DAO/implementations/UserDAOImpl.java

```diff
@@ -74,16 +74,12 @@ public String deleteUser(long userId) throws DatabaseErrorException {
 
 
     @Override
-    public List<User> getUsersByPage(int pageId, int total) {
+    public List<User> getUsersByPage(int pageId, int usersByPage) {
```

Comment:
if `usersByPage` is a number of users on the page, you should give corresponding name

Reply:
Changed it to `usersOnPage`